utils/random.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Utilities for handling random data generation
+ */
+
+import * as t from "io-ts";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { NonEmptyString, PatternString } from "italia-ts-commons/lib/strings";
+import * as randomstring from "randomstring";
+
+const UPPERCASED_LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWYZ";
+const LOWERCASE_LETTERS = "abcdefghijklmnopqrstuvwyz";
+const NUMBERS = "0123456789";
+const SYMBOLS = "!£$%&()?+@=€";
+
+// at least 10 characters, at least one symbol one uppercase, one lowercase, one number
+const STRONG_PASSWORD_PATTERN =
+  "(?=.{10,})(?=.*[!£$%&()?+@=€].*)(?=.*[a-z].*)(?=.*[A-Z].*)(?=.*[0-9].*)";
+
+export const StrongPassword = t.intersection([
+  PatternString(STRONG_PASSWORD_PATTERN),
+  NonEmptyString
+]);
+export type StrongPassword = t.TypeOf<typeof StrongPassword>;
+
+const shuffleString = (str: string): string => {
```

Comment:
see the comment above, remove this module and just generate a 18 character string from the needed charset